index.js

```diff
@@ -74,3 +74,13 @@ exports.getStatusText = function(statusCode) {
     throw new Error("Status code does not exist: " + statusCode);
   }
 };
+
+exports.getStatusCode = function(statusText) {
+  for (key in statusCodes) {
+    if (statusCodes[key] === statusText) {
+      return parseInt(key, 10);
```

Comment:
Do we want to be more defensive here and check if parseInt is a number?

Reply:
I think it's fine. Lets assume `key` is always some Number (status code) coerced into a string.